TKGame/UI/MainMenu.cs

```diff
@@ -64,34 +70,26 @@ public static void Initialize(Desktop desktop, TKGame game)
             {
                 TKGame.paused = false;
                 DisableMainMenu();
-                TKGame.SwitchToGameplayMenu();
+                MenuHandler.SwitchToMenu(MenuHandler.MenuState.GAME_MENU);
             };
 
-            exitButton.TouchDown += (sender, eventArgs) => 
+            exitButton.TouchDown += (sender, eventArgs) =>
             {
-                LevelEditor.SaveStageDataToJSON(TKGame.levelComponent.GetCurrentStage(), "auto_saved_stage_data");
```

Comment:
This was the main way to save a stage, perhaps an exit and save button would be nice assuming that ESC opens up the pause menu for the player to click exit and save ( or just for us doesn't *really* matter

Reply:
yeah i was planning on maybe implementing a pause menu when we press paused that just has an exit button instead of just pressing esc to exit. i can maybe cook that up real quick before we merge this